src/components/Sidebar.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import clsx from 'clsx';
+
+
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
```

Comment:
For core stuff you can do:
import { Compnent1, Component2, Component3 } from '@material-ui/core'
might save some space! :)